book/from_flat_php_to_symfony2.rst

```diff
@@ -705,7 +737,7 @@ for example, the list template written in Twig:
         </ul>
     {% endblock %}
 
-The corresponding ``layout.html.twig`` template is also easier to write:
+The corresponding ``layout.html.php`` template is also easier to write in Twig:
```

Comment:
Looked again. On the 718th line it should be `list.html.php` and then on the 740th line it should be `layout.html.php` but without "corresponding" which adds a bit of confusion.
The chapter flow is:
1) create list and layout .php file
2) rewrite them in *.html.php
3) and finally rewrite them in *.html.twig